tests/python/relay/test_check_kind.py

```diff
@@ -29,6 +29,16 @@ def test_func_kind():
     tf = relay.FuncType(arg_types, ret_type, type_params, type_constraints)
     assert check_kind(tf)
 
+def test_type_relation_kind():

```

Comment:
Would it be appropriate also to have test cases where `TypeRelation`s are used in the constraints field of function types, or is that aspect of the design too much in flux?

Reply:
That should be fine